common/shlibs

```diff
@@ -2017,11 +2017,11 @@ libFcitxQt5WidgetsAddons.so.1 libfcitx-qt5-1.2.1_1
 libfcitx-qt5.so.0 libfcitx-qt5-0.1.3_1
 liblastfm.so.1 liblastfm-1.0.9_1
 liblastfm_fingerprint.so.1 liblastfm-1.0.9_1
-libdruntime-ldc-debug-shared.so.83 ldc-runtime-1.13.0_1
-libdruntime-ldc-shared.so.83 ldc-runtime-1.13.0_1
-libphobos2-ldc-shared.so.83 ldc-runtime-1.13.0_1
-libphobos2-ldc-debug-shared.so.83 ldc-runtime-1.13.0_1
-libldc-jit.so.83 ldc-runtime-1.13.0_1
+libdruntime-ldc-debug-shared.so.89 ldc-runtime-1.19.0_1
```

Comment:
A soname bump means that all packages that link against the library need to be revbumped to be rebuilt against the new version. In this case the packages affected are:

```
gtkd
tilix
```

According to `xrevshlib`.